.github/workflows/wizcli-dirscan.yaml

```diff
@@ -0,0 +1,32 @@
+name: 'wiz-cli'
+on: [pull_request]
+
+jobs:
+  wiz-cli-dir-scan:
+    name: 'Wiz-cli Directory Scan'
+    runs-on: ubuntu-latest
+    env:
+      SCAN_PATH: "./" # Set the relative path in the repo to scan
+      # POLICY: "Default Vulnerabily policy" # Set the desired Wiz CLI policy to use, BLANK FOR ALL
+
+    # Use the Bash shell regardless whether the GitHub Actions runner is ubuntu-latest, macos-latest, or windows-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    # Checkout the repository to the GitHub Actions runner
+    - name: Check out repository
+      uses: actions/checkout@v2
```

Comment:
**issue:** 
please update action version and make sure you checkout current ref

like that

        with:
          fetch-depth: 0
          ref: ${{ github.head_ref }}